components/permissions/permission_lifetime_pref_names.h

```diff
@@ -0,0 +1,18 @@
+/* Copyright (c) 2021 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_COMPONENTS_PERMISSIONS_PERMISSION_LIFETIME_PREF_NAMES_H_
+#define BRAVE_COMPONENTS_PERMISSIONS_PERMISSION_LIFETIME_PREF_NAMES_H_
+
+namespace permissions {
+namespace prefs {
+
+extern const char kPermissionLifetimeRoot[];
```

Comment:
nit: we recommend to use `constexpr` for string constants. With it, we don't need to create cc file.